backend_tools/test_db_provider.ts

```diff
@@ -0,0 +1,8 @@
+import DbProvider from "./db_provider";
+
+class DbTestProvider extends DbProvider {
+    user_path: string = "TestUsers"
```

Comment:
Should these potentially be changed to be production paths?
```suggestion
    user_path: string = "Users"
```

Reply:
No they shouldn't
These paths are for testing purposes only.

Our test scripts will make use of the db provider for testing